src/utils/checkUpdateAvailable.ts

```diff
@@ -0,0 +1,24 @@
+import chalk from 'chalk';
+import semver from 'semver';
+import fetch from 'node-fetch';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const pkg = require('../../package.json');
```

Comment:
As far as I know, `import` instead of `require` works for `json`.

```
import { name, version } from '../../package.json';
```

Reply:
This doesn't work because the `package.json` is outside the `rootDir` we set in the `tsconfig`. Do you know a way around this?

Reply:
Yes, but nothing worth the hassle.